From 477c4e891bd4d9e41fcbaa07cd1a37c0975df5cc Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Wed, 4 May 2022 16:32:48 -0600
Subject: [PATCH] zck: declare write_data as static

This needs to be declared static to avoid a symbol conflict:
io.c:(.text+0xe8): multiple definition of `write_data'; src/zck.p/zck.c.o:zck.c:(.text+0x1f4): first defined here

Fixes:
 - http://autobuild.buildroot.net/results/225/22590a7038a40da3700d56c1f82f7dc74225702a

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[james.hilliard1@gmail.com: backport from upstream commit
477c4e891bd4d9e41fcbaa07cd1a37c0975df5cc]
---
 src/zck.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/zck.c b/src/zck.c
index 7727479..83b2630 100644
--- a/src/zck.c
+++ b/src/zck.c
@@ -159,7 +159,7 @@ static error_t parse_opt (int key, char *arg, struct argp_state *state) {
 
 static struct argp argp = {options, parse_opt, args_doc, doc};
 
-void write_data(zckCtx *zck, char *data, ssize_t in_size) {
+static void write_data(zckCtx *zck, char *data, ssize_t in_size) {
     if(zck_write(zck, data, in_size) < 0) {
         LOG_ERROR("%s", zck_get_error(zck));
         exit(1);
-- 
2.25.1

